src/routes/_index.tsx

```diff
@@ -306,7 +317,7 @@ function RepositoryEntry({ repo }: { repo: SerializeFrom<Repository> }): ReactNo
           to={`/?${new URLSearchParams({
             path: repo.path
           }).toString()}`}
-          prefetch="intent"
+          prefetch="none"
```

Comment:
None is default, so you can just remove the prefetch.

Reply:
ah oke. I like having it explicit though, so we don't accidentally set it to something else in the future